astropy/io/fits/hdu/hdulist.py

```diff
@@ -510,6 +510,25 @@ def fileinfo(self, index):
 
         return output
 
+    def __copy__(self):
+        """
+        Return a shallow copy of the object.
+
+        Returns
+        -------
+        self : HDUList
```

Comment:
shouldn't be `self` because it doesn't return `self`. What about `copy : HDUList`? Also please put single backticks around `HDUList`.